ext/intl/formatter/formatter_format.c

```diff
@@ -103,9 +103,15 @@ PHP_FUNCTION( numfmt_format )
 			}
 			INTL_METHOD_CHECK_STATUS( nfo, "Number formatting failed" );
 			break;
-
+		case FORMAT_TYPE_CURRENCY:
+			if (getThis()) {
+				zend_argument_value_error(2, "cannot use NumberFormatter::TYPE_CURRENCY constant, use formatCurrency() method instead");
+			} else {
+				zend_argument_value_error(3, "cannot use NumberFormatter::TYPE_CURRENCY constant, use numfmt_format_currency() function instead");
+			}
```

Comment:
IMO, referring to `NumberFormatter::formatCurrency()` in both cases should be sufficient. After all, they can be used interchangeably, and if it was for me, we would deprecate the procedural API anyway.